lib/timeline/Core.js

```diff
@@ -677,6 +677,46 @@ Core.prototype.getWindow = function() {
 };
 
 /**
+ * Zoom in the window such that given time is centered on screen.
+ * @param {Number} percentage
```

Comment:
maybe an additional explanation? That the percentage of the min-max-timespan. Percentage must be be [0..1]